net/ola/Makefile

```diff
@@ -9,12 +9,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=ola
-PKG_VERSION:=0.10.8
-PKG_RELEASE:=4
+PKG_VERSION:=0.10.9
+PKG_RELEASE:=1
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
```

Comment:
no need to change these for a version bump, please do this in a separate commit, if at al

Reply:
This commit is picked from the master branch as-is.